libexec/scoop-bucket.ps1

```diff
@@ -19,19 +19,48 @@
 #     scoop bucket known
 param($cmd, $name, $repo)
 
-. "$psscriptroot\..\lib\core.ps1"
-. "$psscriptroot\..\lib\buckets.ps1"
-. "$psscriptroot\..\lib\help.ps1"
+. "$PSScriptRoot\..\lib\core.ps1"
+. "$PSScriptRoot\..\lib\buckets.ps1"
+. "$PSScriptRoot\..\lib\help.ps1"
 
 reset_aliases
 
 $usage_add = "usage: scoop bucket add <name> [<repo>]"
 $usage_rm = "usage: scoop bucket rm <name>"
 
-switch($cmd) {
+function list_buckets {
+    $buckets = @()
+
+    foreach ($bucket in Get-LocalBucket) {
+        $source = Find-BucketDirectory $bucket -Root
+        $manifests = (
+            Get-ChildItem "$source\bucket" -Force -Recurse -ErrorAction SilentlyContinue |
+            Measure-Object | Select-Object -ExpandProperty Count
+        )
+        $updated = 'N/A'
+        if (Test-Path (Join-Path $source '.git')) {
+            $updated = git_cmd -C "`"$source`"" log --format='%ch' -n 1
+            $source = git_cmd -C "`"$source`"" config remote.origin.url
+        }
+        if ($source.StartsWith((Get-PSProvider 'FileSystem').Home)) {
+            $source = $source.Replace((Get-PSProvider 'FileSystem').Home, '~')
+        }
```

Comment:
Use the `friendly_path` function for this: https://github.com/ScoopInstaller/Scoop/blob/develop/lib/core.ps1#L381